fossunited/stack/doctype/stack/stack.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024, Frappe x FOSSUnited and contributors
+# For license information, please see license.txt
+
+# import frappe
+from frappe.model.document import Document
+
+
+class Stack(Document):
+    # begin: auto-generated types
+    # This code is auto-generated. Do not modify anything in this block.
+
+    from typing import TYPE_CHECKING
+
+    if TYPE_CHECKING:
+        from frappe.types import DF
+
+        category: DF.Data
+        icon: DF.AttachImage
+        name1: DF.Data
```

Comment:
nit - why does this have to be `name1`? looks like there was some issue when creating the doctype.

Reply:
could we go with `project_name` instead of `name1` - that's more descriptive.

Reply:
let's go with `title`